terraform/aks-rbac.tf

```diff
@@ -1,47 +1,49 @@
-resource "azurerm_resource_group" "k8s" {
-    name     = "${var.resource_group_name}"
-    location = "${var.location}"
-}
-
-resource "azurerm_kubernetes_cluster" "k8s" {
-    name                = "${var.cluster_name}"
-    location            = "${azurerm_resource_group.k8s.location}"
-    resource_group_name = "${azurerm_resource_group.k8s.name}"
-    dns_prefix          = "${var.dns_prefix}"
-    kubernetes_version  = "${var.kubernetes_version}"
-
-
-    linux_profile {
-        admin_username = "ubuntu"
-
-        ssh_key {
-        key_data = "${file("${var.ssh_public_key}")}"
-        }
-    }
-
-    agent_pool_profile {
-        name            = "default"
-        count           = "${var.agent_count}"
-        vm_size         = "Standard_DS2_v2"
-        os_type         = "Linux"
-        os_disk_size_gb = 30
-    }
-
-    service_principal {
-        client_id     = "${var.client_id}"
-        client_secret = "${var.client_secret}"
-    }
-
-    tags {
-        Environment = "Development"
-    }
-
-    role_based_access_control {
-        azure_active_directory {
-            server_app_id     = "${var.rbac_server_app_id}"
-            server_app_secret = "${var.rbac_server_app_secret}"
-            client_app_id     = "${var.rbac_client_app_id}"
-            tenant_id         = "${var.tenant_id}"
-        }
-    }
+resource "azurerm_resource_group" "k8s" {
+    name     = "${var.resource_group_name}"
+    location = "${var.location}"
+}
+
+resource "azurerm_kubernetes_cluster" "k8s" {
+    name                = "${var.cluster_name}"
+    location            = "${azurerm_resource_group.k8s.location}"
+    resource_group_name = "${azurerm_resource_group.k8s.name}"
+    dns_prefix          = "${var.dns_prefix}"
+    kubernetes_version  = "${var.kubernetes_version}"
+
+
+    linux_profile {
+        admin_username = "ubuntu"
+
+        ssh_key {
+        key_data = "${file("${var.ssh_public_key}")}"
+        }
+    }
+
+    agent_pool_profile {
+        name            = "default"
+        count           = "${var.agent_count}"
+        vm_size         = "Standard_DS2_v2"
+        os_type         = "Linux"
+        os_disk_size_gb = 30
+    }
+
+    service_principal {
+        client_id     = "${var.client_id}"
+        client_secret = "${var.client_secret}"
+    }
+
+    tags {
+        Environment = "Development"
+    }
+
+    role_based_access_control {
+        enabled = 1
```

Comment:
I think it should be `true` rather than `1`. It works with `Terraform v0.11.14`.